### PR TITLE
update files to match new project structure

### PR DESCRIPTION
**Pull request by NikoBergemann**
adjust to new projectname, assets  path, gradle structure, etc.

took release from v0.12.2

### CODE REVIEW

**Comment by NikoBergemann** [comment]
PROBLEM::: 


tar: UC4-chaincode/src: Cannot mkdir: Permission denied
tar: UC4-chaincode/src: Cannot mkdir: Permission denied
tar: UC4-chaincode/src/main: Cannot mkdir: No such file or directory
tar: UC4-chaincode/src: Cannot mkdir: Permission denied
tar: UC4-chaincode/src/main/java: Cannot mkdir: No such file or directory
tar: UC4-chaincode/src: Cannot mkdir: Permission denied
tar: UC4-chaincode/src/main/java/de: Cannot mkdir: No such file or directory
tar: UC4-chaincode/src: Cannot mkdir: Permission denied
tar: UC4-chaincode/src/main/java/de/upb: Cannot mkdir: No such file or directory
tar: UC4-chaincode/src: Cannot mkdir: Permission denied

**Review by anemonekk — APPROVED** [review]

**Comment by NikoBergemann** [comment]
version handling for chaincode?

**Comment by NikoBergemann** [comment]
seems to work for now https://github.com/upb-uc4/hlf-api/runs/1381000488?check_suite_focus=true

**Review by anemonekk — APPROVED** [review]

File: deploy.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 source ./scripts/util.sh
 
-BRANCH_TAG="develop"
+CHAINCODE_VERSION="v0.12.2"
```
Comment (palkra) [inline deploy.sh:8]: Newest tag as default would be better

File: kind.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+kind: Cluster
```
Comment (palkra) [inline kind.yaml:1]: Why needed?
If needed, please move to assets folder.

File: scripts/installChaincode.sh [hunk 3]
```diff
@@ -26,14 +26,11 @@ source ./scripts/env.sh
...
-mkdir -p $HL_MOUNT/uc4
-wget -c https://github.com/upb-uc4/hlf-chaincode/archive/"$BRANCH_TAG".tar.gz -O - | tar -xz -C $HL_MOUNT/uc4 --strip-components=1
-
-header "Build"
-msg "Building chaincode using gradle"
-pushd $HL_MOUNT/uc4/chaincode
-./gradlew installDist
-popd
+mkdir -p $HL_MOUNT/uc4/assets
+mkdir -p $HL_MOUNT/uc4/UC4-chaincode
```
Comment (palkra) [inline scripts/installChaincode.sh:30]:
```suggestion
mkdir -p $HL_MOUNT/uc4/UC4-chaincode

```

File: deploy.sh (hunk 1 above)
Comment (NikoBergemann) [reply deploy.sh:8]: https://github.com/upb-uc4/hlf-network/issues/101

File: kind.yaml (hunk 2 above)
Comment (NikoBergemann) [reply kind.yaml]: deleted

**Review by palkra — APPROVED** [review]

**Review by jonas-thiele — APPROVED** [review]

**Pull request merged**